test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/OtlpExporterTest.cs

```diff
@@ -147,7 +147,7 @@ public void ToOtlpSpanTest()
                 new KeyValuePair<string, object>("double", 3.14),
 
                 // TODO: update if arrays of standard attribute types are supported
-                new KeyValuePair<string, object>("unknown_attrib_type", new byte[] { 1 }),
+                new KeyValuePair<string, object>("unknown_attrib_type", new bool[] { true }),
```

Comment:
nit: Can probably remove the TODO above this now?

Reply:
sure! :)